features/contract-state/Totals.tsx

```diff
@@ -69,14 +72,37 @@ const Totals = () => {
     const getExchangeLinkToken = exchangeInfo.getExchangeUrl(tokenAddress);
     const exchangeName = exchangeInfo.name;
 
+    const addTokenToMetamask = () => {
+      // Add token to users metamask wallet.
+      if (provider == null) return;
```

Comment:
So you can remove this `provider === null` check now that you already check it right?

Reply:
@chrismaree nit here